concourse/pipelines/deploy.yml

```diff
@@ -327,7 +327,9 @@ jobs:
     - get: govuk-infrastructure
       trigger: true
     - file: govuk-infrastructure/concourse/pipelines/deploy.yml
-      set_pipeline: deploy-apps-test
+      set_pipeline: ((pipeline_name))
+      var_files:
+        - govuk-infrastructure/concourse/pipelines/deploy-parameters.yml
```

Comment:
I wonder if we should replace `var_files` with `instance_vars`? This would enable us to bring up a pipeline for a new workspace, without needing to push a branch to GitHub.

```yaml
instance_vars
      workspace: ((workspace))
      govuk_infrastructure_branch: ((govuk_infrastructure_branch))
```

Apparently `instance_vars` is currently experimental and requires the feature flag `--enable-pipeline-instances` but I think it's worth switching to it, even if we later need to use a config file.

Reply:
I prefer to keep it out, easier for a newcomer to see which file they need to modify without modifying their stuff.

I couldn't see where to pass the flag also easily

Reply:
I think `instance_vars` is new in concourse 7, and we're still on 6. So I think we need to stick with plain ol' vars for now. Might be worth asking autom8 if they'll enable `--enable-pipeline-instances` for us when they upgrade, if we want to get on the bleeding edge though.